wordpress-mobile/WordPress-iOS
# Show/hide nav bar and toolbar when scrolling

Fixes #14244

Review only after #14283 is merged.

This PR adds the feature to hide/show the navigation bar and the toolbar while the user scrolls.

<img src="https://user-images.githubusercontent.com/7040243/84779716-1be59f80-afbb-11ea-998c-3537189ba015.gif" width="300" />

To achieve a more smooth hiding/showing I used the https://github.com/andreamazz/AMScrollingNavbar library. It's a very small and popular library.

The reason I didn't use `hidesBarsOnSwipe` is that the transition is kinda glitchy. Plus, that would require more custom code to hide the toolbar (and apparently not even Apple uses it).

### To test

**Make sure that you have overridden the new reader Feature Flag and it's not using the older Reader (Me -> App Settings -> Debug -> "Reader content displayed in a webview" should be ON)**

1. Open a post in the Reader
2. Scroll up, down, go back to the post lists, open another one, repeat :)

PR submission checklist:
